### PR TITLE
4 acm lab.md matthew kunkel

### LINKED ISSUE #115: 4-ACM Fix Formating
In the ACM Lab, there are large blocks of text with the prime information lost in absurd amounts of text. I wish to add steps and bold clear instructions for the lab, leaving lesser information behind. My plan is to take the text and put it into a bullet list, with the first being the main point, the second bullet being any lines of code or any text related to creating/editing code, then the third being the extra stuff. I would bold the titles and key points, and make the text smaller and italicize the remaining text.

### PR DESCRIPTION
**Pull request by StarDusGamer**
Fixes #115
I have changed the formatting to properly show steps and bolded and bulleted important information. The less important information is greyed out but still available if desired. I made it visually appealing so your eyes wont get lost in large amounts of text.

### CODE REVIEW

**Review by onzfonz — CHANGES_REQUESTED** [review]
So in addition to fixing the issues (you can look at the animation on the assignment page on canvas to go through all of the issues)

Please also make sure that for any of the grayed out parts, you continue using the `>` sign at the beginning of each line so that it becomes easier to recognize what is in gray and what is not.
If you don't understand any of the rules that are causing errors, feel free to reach out.

**Comment by StarDusGamer** [comment]
I have addressed the suggestions and all errors have been fixed

**Comment by StarDusGamer** [comment]
I have added the > for every Gray Bullet

**Review by StarDusGamer — COMMENTED** [review]
Changes Implemented

File: docs/labs/4-ACM.md [hunk 1]
```diff
@@ -33,15 +47,18 @@ We are going to use this library because:
 2. Its structure follows OOP programming well
 3. The way it's laid out will well prepare you to learn other languages and libraries in the future
 
-First,
+## Step 1: Download
+
+1. First,
 download **IntroACMLab.zip**
 from canvas and then import that file as an existing project into Eclipse,
```
Comment (onzfonz) [inline docs/labs/4-ACM.md:54]:
```suggestion
1. First, download **IntroACMLab.zip** from canvas
and then import that file as an existing project into Eclipse,
```

File: docs/labs/4-ACM.md [hunk 2]
```diff
@@ -68,226 +85,230 @@ public class ACMLab extends GraphicsProgram {
...
-- **Imports:** Lines 1-4 contain ```import``` statements:
+To explain what is going on, ***the following is not neccessary but is important to know***
+
+- **Imports:**
+    - Lines 1-4 contain ```import``` statements:
   ```import``` calls are similar to ```#include```s that we had in C++
   in that they help Java figure out what libraries you will be using.
-  One of the nice features of Java
-  is that it allows you to reference a folder of classes all at once,
+    - **The majority of imports we'll use start with "java." or "acm."**
```
Comment (onzfonz) [inline docs/labs/4-ACM.md:94]:
```suggestion
    - **The majority of imports we'll use start with `java.` or `acm.`**
```

File: docs/labs/4-ACM.md [hunk 3]
```diff
@@ -68,226 +85,230 @@ public class ACMLab extends GraphicsProgram {
...
+    **It is good programming practice to declare any number that is not 0, 1, or 2, as a constant**.
+    - We're going to break with that convention for the opening part of the tutorial
+  as we are introducing you to the different calls.
+
+  ***Once you get into the activity, please make new constants***
+
+  > Constants make your program easier to change in the future and easier to read.
+     While there are a couple of extra keywords here that I don't want you to worry about too much yet,
+     the ```final``` keyword is the one that is telling Java
+     that a particular variable can no longer be changed.
```
Comment (onzfonz) [inline docs/labs/4-ACM.md:141]:
```suggestion
  > While there are a couple of extra keywords here that I don't want you to worry about too much yet,
  > the ```final``` keyword is the one that is telling Java
  > that a particular variable can no longer be changed.
```

File: docs/labs/4-ACM.md [hunk 4]
```diff
@@ -68,226 +85,230 @@ public class ACMLab extends GraphicsProgram {
...
+- [Lines](#lines)
+
+### Rectangles
+
+1. *To make a rectangle, create a ```GRect``` and then add it to the **canvas** AKA the program window.*
+
+    > ```new GRect``` takes in four arguments,
+    > the ```x``` and ```y```, and its ```width``` and ```height``` all in pixels.
+
+1. *Let's place a rectangle at ```START_X, START_Y``` with a width of ```200```,
```
Comment (onzfonz) [inline docs/labs/4-ACM.md:288]:
```suggestion
2. *Let's place a rectangle at ```START_X, START_Y``` with a width of ```200```,
```

**Review by onzfonz — CHANGES_REQUESTED** [review]
Please use the add suggestions to batch feature here @StarDusGamer and then if this is done correctly, then we can finish off the review and I can approve on the next round.

**Review by onzfonz — APPROVED** [review]
LGTM

**Pull request merged**